presto-main/src/main/java/com/facebook/presto/operator/ExchangeOperator.java

```diff
@@ -95,6 +96,8 @@ public void noMoreOperators()
     private final PagesSerde serde;
     private ListenableFuture<?> isBlocked = NOT_BLOCKED;
 
+    private final LocalMemoryContext systemMemoryContext;
```

Comment:
Can you please group the variable declarations? THis one should be right after the last `private final` line 96. Then add an empty line between `private final` and `private`